src/baked-pasta-with-broccoli.md

```diff
@@ -0,0 +1,37 @@
+# Baked pasta with broccoli, boiled eggs and scamorza cheese
+
+![Picture](pix/baked-pasta-final.webp)
+
+- ⏲️ Prep time: 15 min
+- 🍳 Cook time: 20~30 min
+- 🍽️ Servings: 4
+
+## Ingredients 
+
+- 500g x pasta
+- 500g x broccoli
+- 4 x eggs 
+- 200g x scamorza cheese
+- 400g x béchamel 
```

Comment:
```suggestion
- 500g pasta
- 500g broccoli
- 4 eggs 
- 200g scamorza cheese
- 400g béchamel 
```

Reply:
Oh I'm sorry, my bad. This is my first pull request ever